docs/components/modeler/web-modeler/process-applications.md

```diff
@@ -0,0 +1,126 @@
+---
+id: process-applications
+title: Process applications
+description: Process applications allow you to deploy multiple related files together in a single bundle.
+---
+
+import EmptyProjectImg from './img/process-applications/empty-project.png'
+import CreateProcessApplicationImg from './img/process-applications/create-process-application.png'
+import FileListImg from './img/process-applications/file-list.png'
+import DeployProcessApplicationImg from './img/process-applications/deploy-process-application.png'
+import RunProcessApplicationImg from './img/process-applications/run-process-application.png'
+import DeployErrorImg from './img/process-applications/deploy-error.png'
+
+<span class="badge badge--cloud">Camunda 8 SaaS only</span>
+
+## Idea and purpose
+
+A process application is a special type of folder in Web Modeler that allows you to work on a set of related files and
+[deploy](#deploy-and-run-a-process-application) them together in a single bundle with just one click.
+This reduces the risk of having a broken deployment at runtime and makes it more convenient to deploy related files.
+
+We advise using a process application if there is one main BPMN process that represents your end-to-end use case and
+additional files the main process depends on – like called supporting processes, DMN decisions, or forms.
+
+## Create a process application
+
+Create a new process application by clicking the **Create process application** button in an empty project or via
+**Create new** > **Process application** on the project or folder page.
+
+<p><img src={EmptyProjectImg} alt="Create a process application from an empty project" /></p>
+
+You will be asked to provide a name for the new process application.
+
+<p><img src={CreateProcessApplicationImg} style={{width: 500}} alt="Create a new process application" /></p>
+
+### Main process
+
+The new process application will contain an empty BPMN diagram with the same name as the process application itself,
+labelled as the main process.
+You can rename the main process diagram any time if you wish.
+
+:::note
+A process application must always have a main process.
+That is why the main process diagram cannot be deleted or moved out of the process application.
```

Comment:
This could be problem with migration. I created https://github.com/camunda/web-modeler/issues/9339 with 🔥 priority